ipn/ipnstate/ipnstate.go

```diff
@@ -81,6 +84,18 @@ type TailnetStatus struct {
 	MagicDNSEnabled bool
 }
 
+// ExitNodeStatus is the exit node used by the local backend.
+type ExitNodeStatus struct {
+	// Congigured is the exit node's ID.
+	Configured tailcfg.StableNodeID
+
+	// Online is whether the node is alive.
+	Online *bool
+
+	//FirstIP is the exit node's first IP address.
```

Comment:
space after //

and clarify that this is the Tailscale IP

Or at this point, now that we have a struct, want to just do `TailscaleIPs []netaddr.IP` like Status and PeerStatus? Then it matches those two.